backend/compound_library/src/graphql.rs

```diff
@@ -0,0 +1,17 @@
+use crate::resolvers::{
+    compound_instances_res::{CompoundInstanceMutation, CompoundInstanceQuery},
+    compound_types_res::{CompoundMutation, CompoundQuery},
+};
+use async_graphql::{EmptySubscription, MergedObject, Schema, SchemaBuilder};
+
+#[derive(Debug, Clone, MergedObject, Default)]
+pub struct Query(CompoundQuery, CompoundInstanceQuery);
+
+#[derive(Debug, Clone, MergedObject, Default)]
+pub struct Mutation(CompoundMutation, CompoundInstanceMutation);
```

Comment:
Imo, `RootMutation` is more descriptive

Reply:
Needed for Apollo federation.

Reply:
Ah, thought you said it was the other way around earlier, oops